packages/ra-core/src/core/createAdminStore.ts

```diff
@@ -15,23 +9,12 @@ interface Window {
 }
 
 interface Params {
-    dataProvider: DataProvider;
-    history: History;
-    authProvider?: AuthProvider;
     customReducers?: any;
-    i18nProvider?: I18nProvider;
     initialState?: InitialState;
-    locale?: string;
 }
 
-export default ({
```

Comment:
while I was debugging the data provide rhooks issue, I messed with the Redux store, and noticed that this method signature was oudated and in dire need for a cleanup